StockServiceFactory/src/main/java/edu/mbrooks/advanced/services/WebStockServiceFactory.java

```diff
@@ -0,0 +1,22 @@
+package edu.mbrooks.advanced.services;
+
+public class WebStockServiceFactory {
+
+    /**
```

Comment:
you could put this all in the same factory class.

Reply:
Yes, I had it that way originally... Thought you wanted something separate...